variables.tf

```diff
@@ -2,6 +2,7 @@
 ## Standard variables
 #######################
 
+# Useless cluster_name & base_domain variables ? TODO remove if it's the case.
```

Comment:
I already posed the same question myself and I think they are not used on this module. I left them last time because in the eventuality they would be needed and for consistency with other modules, but I'm fine if we take them out to avoid confusions (there is no point keeping code that is not used).

Reply:
Yes there are unused variables in many modules, they can be a source of confusion especially when they're required. The initial goal was to have a set of common variables to all modules in `variables.tf` but I don't think it's necessary.